llama_index/vector_stores/postgres.py

```diff
@@ -26,6 +26,7 @@ def get_data_model(
     index_name: str,
     hybrid_search: bool,
     text_search_config: str,
+    set_cache_ok: bool,
```

Comment:
very nit: keep this as `cache_okay` for consistency?

Reply:
done